code/style.css

```diff
@@ -1,6 +1,26 @@
 body {
   margin: 0;
+  padding: 0;
   font-family: "Courier New", Courier, monospace;
+  background-image: url(./assets/hallway.jpeg);
+  background-size: cover;
+  backdrop-filter: blur(9px);
+}
+
+h1 {
+  text-align: center;
+  letter-spacing: 30px;
+  border: 10px double yellowgreen;
+  padding: 50px;
+  margin: 0px 300px 0px 300px;
+  font-size: 60px;
+  color: yellow;
+}
+
+h3,h2 {
```

Comment:
Could "h2" be removed since it's not used in the HTML?

Reply:
👍